components/crud-web-apps/tensorboards/frontend/package.json

```diff
@@ -12,7 +12,9 @@
     "test": "ng test",
     "test:prod": "ng test --browsers=ChromeHeadless --watch=false",
     "lint-check": "ng lint",
-    "lint": "ng lint --fix"
+    "lint": "ng lint --fix",
+    "format:check": "prettier --check 'src/**/*.{js,ts,html,scss,css}' || node scripts/check-format-error.js",
```

Comment:
I see that we don't have this script in TWA, like we do in Katib
https://github.com/kubeflow/katib/tree/master/pkg/new-ui/v1beta1/frontend/scripts

Yet the action didn't fail. Why is that?

Reply:
Oh sorry I missed that. Should I add this script in TWA as well?

Reply:
OK I see what happened. `||` means that the second command (node script in this case) will NOT be run if the first part was successful. But the node script will run if the prettier check fails.

That script is just there to instruct users to run the formatting. Let's include it in this PR

Reply:
ACK! I just added it!